src/zzz/zh/docs/feat_notify.md

```
@@ -9,30 +9,784 @@ title: 功能-通知
+>    - `配置步骤`：详细的配置流程指导
+> 2. 所有参数都标注了是否必填
+> 3. 界面配置项与配置文件参数一一对应，方便查找
+> 4. 本文档基于公开资料和代码实现整理， 并非所有内容都已100%证实， 如果你发现问题，欢迎前往[频道](https://pd.qq.com/s/f4j1v9p0d) 或 ISSUE 区反馈.
+
+
+## 目录
+- [基础说明](#基础说明)
+- [通知方式](#通知方式)
+- [配置步骤](#配置步骤)
+- [详细配置参数](#详细配置参数)
+- [故障自助排查](#故障自助排查)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)